common/src/app/common/schema.cljc

```diff
@@ -240,6 +240,7 @@
   (when-not ^boolean (-validate s value)
     (let [hint    (d/nilv dm/*assert-context* "check error")
           explain (-explain s value)]
+      (println (humanize-explain explain))
```

Comment:
is this println necessary?

Reply:
Let's see. The lib-penpot packaging had a problem, that if a schema error occurred there, the exception trace was minified and illegible. So Andrey added this println to be able to see the cause.

I'l remove it in a separated commit for now. Andrey wants to review it to try to solve properly. But I've removed it in a separated commit, to be able to locate it easier if we need it again until then.